tests/test_io_utils.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+import fsspec
+import pandas as pd
+
+import ecoscope
+
+
+def test_download_file_github_csv():
+    ECOSCOPE_RAW = "https://raw.githubusercontent.com/wildlife-dynamics/ecoscope/master"
+    output_dir = "tests/test_output"
+    ecoscope.io.download_file(
+        f"{ECOSCOPE_RAW}/tests/sample_data/vector/movbank_data.csv",
+        os.path.join(output_dir, "download_data.csv"),
+        overwrite_existing=True,
+    )
+
+    data = pd.read_csv(os.path.join(output_dir, "download_data.csv"))
+    assert len(data) > 0
+
+
+def test_download_file_gdrive():
+    output_dir = "tests/test_output"
+    ecoscope.io.download_file(
+        "https://drive.google.com/uc?export=download&id=1-AQ9_oacUCcAaiZ6SWU77hZWp1oArQw6",
+        os.path.join(output_dir, "download_data.csv"),
+        overwrite_existing=True,
+    )
+
+    data = pd.read_csv(os.path.join(output_dir, "download_data.csv"))
+    assert len(data) > 0
+
+
+def test_download_file_gdrive_zip():
+    output_dir = "tests/test_output"
+    ecoscope.io.download_file(
+        "https://drive.google.com/uc?export=download&id=1YNQ6FBtlTAxmo8vmK59oTPBhAltI3kfK",
+        output_dir,
+        overwrite_existing=True,
+    )
+
+    data = pd.read_csv(os.path.join(output_dir, "movbank_data.csv"))
+    assert len(data) > 0
+
+
+def test_download_file_dropbox_json():
+    URL = "https://www.dropbox.com/scl/fi/qaw3krcsnot69x94mdfxy/config.json?rlkey=zdmipl2la7rplgl218vc13end&dl=1"
```

Comment:
Think we should include code to automatically change dl=0 to dl=1 to help out users from having to modify a dropbox url themselves.